index.html

```diff
@@ -160,12 +160,12 @@ <h2>
         <dd>
           An event generated by the user agent as a result of user interaction
           processed by the user agent to interact with the page. e.g.
-          <code>click</code>, but not <code>mousemove</code>. Engagement
-          gestures are any events included in the definition of being
-          <a>triggered by user activation</a> with the addition of
-          <code>keypress</code> and <code>keyup</code>. Note that operating
-          system level accessibility and internationalization features may
-          intercept gestures before the user agent processes them for
+          <code>click</code>, but not <code>mousemove</code>.  Engagement
+          gestures are defined by the events in the list of <a>activation
```

Comment:
This is good, but I was expecting this also to be part of `.requestPointerLock()` too... that is, a `NotAllowedError` should be thrown when this check happens.

Reply:
We don't support fully the error list and whatnot. We only have this pointerlockerror event at this point. But that will be a good addition to the new promise that has more detailed errors that we have coming up.

Reply:
Ok, sounds good. Treating this as an editorial change.